hs_geo_raster_resource/serialization.py

```diff
@@ -1,13 +1,10 @@
 import xml.sax
```

Comment:
Are these changes in this file are all pylint related and no actual code change?